### DISCUSSION TITLE
Get NodeId of custome Object Type by BrowseName

### DISCUSSION THREAD

**Discussion by janfelixklein** [FreeOpcUa/opcua-asyncio#748] [General]
Hello everyone,

I have the following problem:
After loading a custom nodeset, I would like to retrieve the nodeid of one of the custom object types by name. Is there any implemented way to do so? I didn't really find something to achieve that.
`asyncua.ua.ObjectIds` would allow to to this for the core nodeset.

Best regards,

Jan-Felix

**Comment by oroulet** [comment]
get_child() uses the browse name which is often similar to the name of the objects. but you need to know the namespace. All objects in ObjectIds are in namespace 0
